lib/content._coffee

```diff
@@ -6,21 +6,18 @@ redis = require 'redis'
 URL = require 'url'
 Worker = require './worker'
 
+pkgcloud = require('pkgcloud')
+cloudClient = pkgcloud.storage.createClient
```

Comment:
Btw, looks like we have this same code in `worker._coffee`, and I called it `storageClient` there.

https://github.com/zoomhub/zoomhub/blob/cloud-files/lib/worker._coffee#L39-L45

Doesn't have to be for this PR, but maybe at some point it'll make sense to de-dupe this with a util, since otherwise we'll also need to duplicate stream-handling logic etc. (which you're asking about below).